overlord/ifacestate/helpers.go

```diff
@@ -26,6 +26,10 @@ import (
 	"github.com/snapcore/snapd/interfaces"
 	"github.com/snapcore/snapd/interfaces/backends"
 	"github.com/snapcore/snapd/interfaces/builtin"
+	"github.com/snapcore/snapd/interfaces/dbus"
```

Comment:
This part of the diff is no longer needed (the entire helpers.go file change), this got refactored. Just appyl:
```
diff --git a/overlord/ifacestate/helpers.go b/overlord/ifacestate/helpers.go
index cd0c7d1..d0690f2 100644
--- a/overlord/ifacestate/helpers.go
+++ b/overlord/ifacestate/helpers.go
@@ -26,10 +26,6 @@ import (
        "github.com/snapcore/snapd/interfaces"
        "github.com/snapcore/snapd/interfaces/backends"
        "github.com/snapcore/snapd/interfaces/builtin"
-       "github.com/snapcore/snapd/interfaces/dbus"
-       "github.com/snapcore/snapd/interfaces/mount"
-       "github.com/snapcore/snapd/interfaces/seccomp"
-       "github.com/snapcore/snapd/interfaces/udev"
        "github.com/snapcore/snapd/logger"
        "github.com/snapcore/snapd/overlord/snapstate"
        "github.com/snapcore/snapd/overlord/state"
@@ -216,7 +212,3 @@ func getConns(st *state.State) (map[string]connState, error) {
 func setConns(st *state.State, conns map[string]connState) {
        st.Set("conns", conns)
 }
-
-var securityBackends = []interfaces.SecurityBackend{
-       &seccomp.Backend{}, &dbus.Backend{}, &udev.Backend{}, &mount.Backend{},
-}
```